src/string.cr

```diff
@@ -4305,23 +4519,27 @@ class String
     char_index
   end
 
-  def clone
+  # Returns `self`.
+  def clone : String
     self
   end
 
-  def dup
+  # :ditto:
+  def dup : String
     self
   end
 
+  # :ditto:
```

Comment:
That should be the other way around - starting with documented `to_s(io)` and having next `to_s` with `:ditto:`

Reply:
No. It's mean ditto dup i.e ditto clone i.e `Returns self.`